drivers/dai/intel/dmic/dmic.c

```diff
@@ -607,18 +607,16 @@ static void dai_dmic_start(struct dai_intel_dmic *dmic)
 		}
 	}
 
-#ifndef CONFIG_SOC_SERIES_INTEL_ACE
 	/* Clear soft reset for all/used PDM controllers. This should
 	 * start capture in sync.
 	 */
 	for (i = 0; i < CONFIG_DAI_DMIC_HW_CONTROLLERS; i++) {
 		dai_dmic_update_bits(dmic, base[i] + CIC_CONTROL,
-				     CIC_CONTROL_SOFT_RESET_BIT, 0);
```

Comment:
@singalsu can you clarify your comment? it's not clear to me if you are suggesting another solution and based on what?

If the blobs should be changed that's not a zephyr change but a topology plugin change to handled by @juimonen

Reply:
@singalsu I tried below diff instead of second commit, but it didn't work. I have only zero input. Can you show me what I missed?

```diff
--- a/drivers/dai/intel/dmic/dmic.c
+++ b/drivers/dai/intel/dmic/dmic.c
@@ -604,12 +604,20 @@ static void dai_dmic_start(struct dai_intel_dmic *dmic)
                switch (dmic->dai_config_params.dai_index) {
                case 0:
                        dai_dmic_update_bits(dmic, base[i] + FIR_CONTROL_A,
+#ifdef CONFIG_SOC_SERIES_INTEL_ACE
+                                            FIR_CONTROL_A_START_BIT | FIR_CONTROL_A_PERIODIC_START_EN_BIT,
+#else
                                             FIR_CONTROL_A_START_BIT,
+#endif
                                             FIR_CONTROL_A_START(fir_a));
                        break;
                case 1:
                        dai_dmic_update_bits(dmic, base[i] + FIR_CONTROL_B,
+#ifdef CONFIG_SOC_SERIES_INTEL_ACE
+                                            FIR_CONTROL_B_START_BIT | FIR_CONTROL_B_PERIODIC_START_EN_BIT,
+#else
                                             FIR_CONTROL_B_START_BIT,
+#endif
                                             FIR_CONTROL_B_START(fir_b));
                        break;
                }
```